docs/content/error/$compile/noctrl.ngdoc

```diff
@@ -0,0 +1,68 @@
+@ngdoc error
+@name $compile:noctrl
+@fullName Controller is required.
+@description
+
+When using the `bindToController` feature of AngularJS, a directive is required
+to have a Controller, in addition to a controller identifier.
```

Comment:
Same as above.
<sub>
BTW, I like _"controller identifier"_ (or _"controller alias"_) better than _"`controller as` identifier"_, as the latter might be misunderstood as implying that a `controllerAs` property is mandatory (which is not the case).
Not a big issue though, since the examples below make it clear.
</sub>